app/views/admin/dashboard/dashboard.html.slim

```diff
@@ -2,7 +2,7 @@
 
 .block
   section.section.recent-projects.projects-grid
-    = render "recent_projects_grid_definition", grid: @recent_projects_grid
+    = render partial: "recent_projects_grid_definition", locals: { person: @person, grid: @recent_projects_grid }
```

Comment:
Can we please do spaces inside {} for procs, but not hashes? I find it's a good way of indicating one vs the other and it's the convention on this project I think.